tests/compile-fail/repeat2.rs

```diff
@@ -1,3 +1,5 @@
+// ignore-test
```

Comment:
Takes too long

Reply:
Is it possible to make the array smaller and still get the failure?

If not I'd vote for just removing the test instead.

Reply:
fun fact: we don't have any limits except stack frame depth anymore. This test won't ever trigger the error